CONTRIBUTING.md

```diff
@@ -18,8 +18,8 @@ or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any addi
 Pull requests are welcome.
 1. Fork the repository.
 2. Make and test your changes (see Developer Guide below).
-3. Run `python -m black .` to format the code (`python -m pip install black` if necessary).
-4. Run `python -m pylint src` and check for no new errors or warnings (`python -m pip install pylint` if necessary).
+3. Run `poetry run black sarif` to format the code.
+4. Run `poetry run pylint sarif` and check for no new errors or warnings.
```

Comment:
Potentially worth adding a request to run `poetry update` here, to ensure that we're on the latest packages to keep us up to date with CVE fixes etc.